### PR TITLE
Delete search icon; instead show empty search input when search inactive.

### PR DESCRIPTION
**Pull request by BugsNash**
- Delete search icon and instead show empty search input field when search is inactive.
- Move tooltip from search icon to a placeholder in search input field. This will be changed to ‘Find’ as per mocks in future when UI text updates are made with translations.
- Change layout to use flex box and center items instead of hard coding padding (this results in a slight shift in the elements in the header).
- Clean up code touched.

### CODE REVIEW

File: css/app.css [hunk 1]
```diff
@@ -409,22 +388,30 @@ header.search-active #search-button {
...
   border: 0;
-  border-left: 1px solid #DDD;
-  transition: all 200ms ease-in-out;
   background-position: 50% 50%;
   background-repeat: no-repeat;
   background-size: 21px;
   width: 0;
   height: 30px;
-  opacity: 0;
+  padding: 0;
```
Comment (E-T-) [inline css/app.css]: Re-arrange this block so that it's in alphabetical order.

File: js/controllers/hotkeys.js [hunk 2]
```diff
@@ -31,8 +31,11 @@ function HotkeysController(windowController, tabs, editor, settings, analytics)
 };
 
 /**
+ * Handles hotkey combination if present in keydown event.
  * Some hotkeys are handled by CodeMirror directly. Among them:
  * Ctrl-C, Ctrl-V, Ctrl-X, Ctrl-Z, Ctrl-Y, Ctrl-A
+ * @param {Event} e The keydown event
```
Comment (E-T-) [inline js/controllers/hotkeys.js]: `@param {!Event} e The keydown event`

The Event should never be non-null so I think you should have the `!` at the beginning.

File: js/controllers/hotkeys.js [hunk 3]
```diff
@@ -46,7 +49,7 @@ HotkeysController.prototype.onKeydown_ = function(e) {
         return false;
 
       case this.KEY.F:
-        $('#search-button').click();
+        document.getElementById('search-input').click();
```
Comment (E-T-) [inline js/controllers/hotkeys.js]: Part of me wonders whether you should use `focus()` instead.

File: css/app.css (hunk 1 above)
Comment (BugsNash) [reply css/app.css]: Done

File: js/controllers/hotkeys.js (hunk 2 above)
Comment (BugsNash) [reply js/controllers/hotkeys.js]: Done. Another thing I need to get better at spotting!

File: js/controllers/hotkeys.js (hunk 3 above)
Comment (BugsNash) [reply js/controllers/hotkeys.js]: Good call. I'll change the listener to look for focus too

**Pull request merged**